buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -193,6 +195,30 @@ class BeamModulePlugin implements Plugin<Project> {
     String tag = null // Sets the image tag (optional).
   }
 
+  // A class defining the configuration for PortableValidatesRunner.
+  class PortableValidatesRunnerConfiguration {
+    // Task name for validate runner case.
+    String name = 'validatesPortableRunner'
+    // Fully qualified JobServerClass name to use.
+    String jobServerDriver
+    // A string representing the jobServer Configuration.
+    String jobServerConfig
+    // Categories for tests to run.
+    Closure testCategories = {
+      includeCategories 'org.apache.beam.sdk.testing.ValidatesRunner'
+      excludeCategories 'org.apache.beam.sdk.testing.FlattenWithHeterogeneousCoders'
```

Comment:
Should these default excludes apply to _all_ runners? If not, I think it makes sense to put this on each runner to specify.

Reply:
Yes, the exclusions here are not supported by fnexecution at the moment hence they apply to all portable validate runners.
As we support more functionality to portability layer, we will remove these exclusions.

Reply:
OK, that makes sense.